src/jbuild

```diff
@@ -30,8 +30,10 @@
  ((name        ocaml_parsing)
   (public_name ocaml_parsing)
   (wrapped true)
-  (libraries 
-    (core
+  (js_of_ocaml ())
```

Comment:
Can we get rid of this line? This is only needed when one wants to produce JavaScript, right?

Reply:
@loxs: Thanks for the PR!